Src/library/memory_analyzer/ma_object_snapshot_mediator.e

```diff
@@ -272,10 +278,13 @@ feature {NONE} -- Implementation
 		do
 			from
 				l_data := grid_data
+				if l_data = Void then
```

Comment:
Looks to me that the whole loop should be surrounded by `if l_data /= Void then`. We don't even need the else clause here since the precondition will caught it.